lib/promoted/ruby/client.rb

```diff
@@ -1,13 +1,12 @@
+require "promoted/ruby/client/async_http_client"
 require "promoted/ruby/client/version"
-require 'faraday'
-require 'json'
 
 module Promoted
   module Ruby
     module Client
 
       DEFAULT_DELIVERY_TIMEOUT_MILLIS = 3000
-      DEFAULT_METRICS_TIMEOUT_MILLIS = 250
+      DEFAULT_METRICS_TIMEOUT_MILLIS = 3000
```

Comment:
Ah, true.  The timeouts are flipped.  We should have shorter timeouts for Delivery API.

Reply:
The unit of timeout we have to work with here is in seconds, not millis, though.

Reply:
So bare minimum is 1000 millis, or else the request never even gets sent in my testing.